packages/cli/README-source.md

```diff
@@ -309,7 +309,7 @@ When a user authenticates to your application through Zapier, a "connection" is
 
 Useful if your app requires two pieces of information to authenticate: `username` and `password`, which only the end user can provide. By default, Zapier will do the standard Basic authentication base64 header encoding for you (via an automatically registered middleware).
 
-> Example App: Check out https://github.com/zapier/zapier-platform/tree/master/example-apps/basic-auth for a working example app for basic auth.
+> To create a new integration with basic authentication, run `zapier init [your app name]` and select "basic-auth" from the list that appears. You can also review an example of that code [here](https://github.com/zapier/zapier-platform/tree/master/example-apps/basic-auth).
```

Comment:
For each of these, you can provide the flag in the command:

> `zapier init [your app name] --template basic-auth`

So I'd include that command in full so people can just paste it.

Reply:
Good shout! Adding those now.